.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.json]
+indent_size = 2
```

Comment:
This currently applies to 2 out of 3 `.json` files. Will follow up with an adjustment to `composer.json`, for consistency, if there are no objections.